css/style.css

```
@@ -0,0 +1,355 @@
+/* fonts */
+@font-face {
+  font-family: "PT Sans";
+  src: url("../fonts/ptsans.woff2") format("woff2"),
+  url("../fonts/ptsans.woff") format("woff");
```

можно немного подформатировать:
```suggestion
  src: url("../fonts/ptsans.woff2") format("woff2"),
       url("../fonts/ptsans.woff") format("woff");
```